### DISCUSSION TITLE
Communicate with another Fast API project through gunicorn unix socket, without nginx

### DISCUSSION THREAD

**Discussion by xiki808** [fastapi/fastapi#6137] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
bind = 'unix:/var/www/myproject/socket/backend.sock'
workers = 1
worker_class = 'uvicorn.workers.UvicornWorker'
```

### Description

The code example shows my gunicorn configuration for two projects running on the same machine ( using different bind socket path for each project ). I would like to know if it's possible to send requests to unix socket, from a Fast API project 'A' to another Fast API project 'B', without having to setup an Nginx proxy. Reason being is that project 'B' is for internal use, and I don't want any public access to it. I assume if it's possible to request directly the socket, it's a better and cleaner solution than having to manage and authorize requests.

### Operating System

Linux

### FastAPI Version

0.81.0

### Python Version

3.9.2

**Comment by xiki808** [comment, marked as answer]
I managed to sort it out by using httpx.AsyncHTTPTransport

```
async with AsyncClient(transport=AsyncHTTPTransport(uds="/var/opt/iowetea-worker/socket/worker.sock")) as client:
                response = await client.get("http://localhost/api/websocket/test")
```

**Comment by tiangolo** [comment]
Thanks for reporting back and closing the issue 👍
